cmd/agree/agree.go

```diff
@@ -192,24 +194,35 @@ var userAnswer = func() (string, error) {
 	return bufio.NewReader(os.Stdin).ReadString('\n')
 }
 
-func parseTermRevision(s string) (string, int, error) {
-	fail := func(err error) (string, int, error) {
-		return "", -1, err
+func parseTermRevision(s string) (string, string, int, error) {
```

Comment:
This will return a nil error for terms like `/1', '//7', etc. Consider exposing the term name validation in `gopkg.in/juju/charm.v6-unstable` (charm package is already imported by romulus) and use it here.